ngraph/frontend/onnx_editor/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "onnx_editor")
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp)
+file(GLOB_RECURSE LIBRARY_PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
+
+# Create named folders for the sources within the .vcproj
+# Empty name lists them directly under the .vcproj
+
+source_group("src" FILES ${LIBRARY_SRC})
+source_group("include" FILES ${LIBRARY_HEADERS})
+source_group("public include" FILES ${LIBRARY_PUBLIC_HEADERS})
+
+# Create shared library
+
+add_library(${TARGET_NAME} SHARED ${LIBRARY_SRC} ${LIBRARY_HEADERS} ${LIBRARY_PUBLIC_HEADERS})
+
+if(COMMAND ie_add_vs_version_file)
+        ie_add_vs_version_file(NAME ${TARGET_NAME}
+                        FILEDESCRIPTION "nGraph ONNX Importer editor")
+endif()
+
+# TODO Add handling ie_faster_build
+
+target_link_libraries(${TARGET_NAME} PRIVATE onnx_common onnx onnx_proto ${Protobuf_LIBRARIES} 
+                                     PUBLIC ngraph)
+
+set(ONNX_EDITOR_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
+set(ONNX_EDITOR_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src)
+set(ONNX_EDITOR_INSTALL_INCLUDE "${NGRAPH_INSTALL_INCLUDE}/ngraph/frontend")
+
+target_include_directories(${TARGET_NAME} SYSTEM PUBLIC $<BUILD_INTERFACE:${ONNX_EDITOR_INCLUDE_DIR}>
+                                                       $<INSTALL_INTERFACE:${ONNX_EDITOR_INSTALL_INCLUDE}>)
+target_include_directories(${TARGET_NAME} PRIVATE ${ONNX_EDITOR_SRC_DIR})
+target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:onnx_common,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
I suppose system is not needed since `onnx_common` is not a 3rd party library. And we already include `onnx_common` headers when we link this library. So, the whole line is not needed

Reply:
removed